src/contractTest/java/uk/gov/hmcts/reform/sscs/utility/BasePactTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.sscs.utility;
+
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+public class BasePactTest {
```

Comment:
(Minor ish) could you have your pact test extend this and have your ContractTestDataProvider be either static or wired into your Pact Test?